data/campaigns/Under_the_Burning_Suns/units/quenoth/Corrupted_Elf.cfg

```diff
@@ -16,7 +16,7 @@
     cost=100
     usage=fighter
     do_not_list=yes
-    description= _ "Imbued with a dark spirit these corrupted elves combine unhuman strength with potent magics to create formidable opponents. Though the elvish body often decays quickly, these abominations are potent weapons of the undead lords."
+    description= _ "Imbued with a dark spirit these Corrupted Elves combine unhuman strength with potent magics to create formidable opponents. Though the elvish body often decays quickly, these abominations are potent weapons of the undead lords."
```

Comment:
Not something to be addressed here, but 'unhuman' sounds odd. I take the original intent to mean 'inhuman' (as in super-human strength) albeit in the context of un-life. Either way, it sounds odd because it's referring to an elf, although the word doesn't change just because it's not referring to a human.